aot/src/test/resources/com/dylibso/chicory/approvals/ApprovalTest.functions10.approved.txt

```diff
@@ -112,490 +112,6 @@ public final class com/dylibso/chicory/$gen/CompiledMachine implements com/dylib
     IRETURN
 }
 
-final class com/dylibso/chicory/$gen/CompiledMachineShaded {
-
-  public final static INNERCLASS java/lang/invoke/MethodHandles$Lookup java/lang/invoke/MethodHandles Lookup
```

Comment:
Looking at the diff I noticed this line, we should not have `MethodHandles` in the generated code.
Seems like it is used only to concatenate Strings in case of an Exception ...